var/spack/repos/builtin/packages/libsonata/package.py

```diff
@@ -19,6 +19,8 @@ class Libsonata(CMakePackage):
     git = "https://github.com/BlueBrain/libsonata.git"
 
     version('develop', git=git, submodules=False)
+    version('v0.0.1', commit='6d7e3929c156812969d9237bd797bf6aba6e50ec', submodules=False)
```

Comment:
sorry, I have overlooked `v` prefix in version here; will change it to `0.0.1` at #364.